src/encoded/static/scss/encoded/modules/_matrix.scss

```diff
@@ -156,7 +156,6 @@ $row-data-header-width: 200px;
 }
 
 .subcategory-row-text {
```

Comment:
These styles are all specific to the mouse matrix, so I would probably change line 158 to be the new lines you added (174 - 176). The regular "subcategory-row-text" does not need the "label-stage" or "label-length" classes because those only correspond to the mouse matrix and `display:flex` so might as well just make it specific to the mouse matrix.

Reply:
Done.

Reply:
Looks good! I probably would have also put `align-items: center` under the subcategory-row-text-mouse class because it's a property that only applies to flexbox display, and I also would have put `line-height: 1.5` under the mouse subcategory because otherwise we're over-writing the line-height on the reference epigenome matrix as well (it has a line-height of 1.15 on production). I was looking at the reference epigenome matrix more and I think there's another change needed there also, it seems like a mouse matrix change might have also eliminated some row height. Maybe lines 184 - 185 need to be moved so they aren't impacting the other matrices? I'm not sure if that's the fix needed, but hopefully.

Reply:
I did the first part (moved line-height and align-item.) For the padding:0 (line 184-185), I moved it unto a class specific for mouse. Prior to the last commit (shah #: 84cd42204), it was causing a very small padding change in other matrices. I moved it so prevent that while not affect mouse matrix itself.